src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -1,4 +1,101 @@
+/*
+ * @(#) WebDriverConfig.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
 package core.selenium;
 
+import core.utils.ConfigFileReader;
+
+/**
+ * WebDriverConfig class.
+ *
+ * @author Cristian Lujan
+ * @version 1.0
+ */
 public class WebDriverConfig {
+    private static final String BROWSER = "browser";
```

Comment:
Verify that its necessary the constants use.

Reply:
done